<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CRM Program</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Tap">
    <property name="title">
     <string>&amp;Tap</string>
    </property>
    <addaction name="action_Client_Manager"/>
    <addaction name="action_Product_Manager"/>
    <addaction name="action_Order_Manager"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="action_New_Chat_Room"/>
    <addaction name="action_ToolBar"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Add"/>
    <addaction name="action_Search"/>
    <addaction name="action_Modify"/>
    <addaction name="action_Remove"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Tap"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Client_Manager"/>
   <addaction name="action_Product_Manager"/>
   <addaction name="action_Order_Manager"/>
   <addaction name="separator"/>
   <addaction name="action_Add"/>
   <addaction name="action_Search"/>
   <addaction name="action_Modify"/>
   <addaction name="action_Remove"/>
   <addaction name="separator"/>
   <addaction name="action_New_Chat_Room"/>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Client_Manager">
   <property name="text">
    <string>&amp;Client Manager</string>
   </property>
  </action>
  <action name="action_Product_Manager">
   <property name="text">
    <string>&amp;Product Manager</string>
   </property>
  </action>
  <action name="action_Order_Manager">
   <property name="text">
    <string>&amp;Order Manager</string>
   </property>
  </action>
  <action name="action_Add">
   <property name="text">
    <string>&amp;Add</string>
   </property>
  </action>
  <action name="action_Search">
   <property name="text">
    <string>&amp;Search</string>
   </property>
  </action>
  <action name="action_Modify">
   <property name="text">
    <string>&amp;Modify</string>
   </property>
  </action>
  <action name="action_Remove">
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="action_New_Chat_Room">
   <property name="icon">
    <iconset theme="phone">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New Chat Room</string>
   </property>
  </action>
  <action name="action_ToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;ToolBar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_ToolBar</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
